src/hydrabus/hydrabus_mode_i2c.c

```diff
@@ -61,11 +61,13 @@ static void init_proto_default(t_hydra_console *con)
 	proto->config.i2c.dev_speed = 1;
 	proto->config.i2c.ack_pending = 0;
 	proto->config.i2c.dev_mode = DEV_MASTER;
+	proto->config.i2c.dev_clock_stretch_timeout = 0;
```

Comment:
I think it will be a good idea to use it by default with a short default timeout set to 10 or more it will help to detect problems with some I2C devices
To be discussed with @Baldanos

Reply:
sounds perfect! once we have a conclusion. I will make the code change.

Reply:
Keep it like that we will change it later if needed

Reply:
Sounds perfect! Please let me know if any other change is needed. I will make it accordingly.